### ISSUE TITLE
Write a blog on Next.js

### ISSUE THREAD

**Issue by debrajrout** [debrajrout/Dev.HFest#46]
Write a blog on Next.js

**Comment by Smitkhobragade** [comment]
I would Like to contribute to this issue.

**Comment by shyam1112** [comment]
I would Like to contribute to this issue. @debrajrout

**Comment by debrajrout** [comment]
> I would Like to contribute to this issue. @debraj
Follow and give a stare in order to merge

**Comment by debrajrout** [comment]
> I would Like to contribute to this issue. @debrajrout

Follow and give a stare in order to merge

**Comment by shyam1112** [comment]
can you suggest where i write this blog please !

**Comment by debrajrout** [comment]
> can you suggest where i write this blog please !

Create a readme file and write there

**Comment by hemantgarg1452** [comment]
Hey, @debrajrout can you please assign this task to me? Currently, I'm working on Next.js projects so I've better things to contribute.

**Comment by debrajrout** [comment]
> Hey, @debrajrout can you please assign this task to me? Currently, I'm working on Next.js projects so I've better things to contribute.

Yes you can but you have to maintain the proper folder structure and give a star

**Comment by hemantgarg1452** [comment]
Okay got your points.
